jupyter_server/serverapp.py

```diff
@@ -951,7 +951,9 @@ def _default_allow_remote(self):
         help=_("Extra keyword arguments to pass to `get_secure_cookie`."
              " See tornado's get_secure_cookie docs for details.")
     )
-    ssl_options = Dict(config=True,
+    ssl_options = Dict(
+            allow_none=True,
```

Comment:
You are now saying that this can be `None`, but the code below does not seem to really allow for it? (I think some code paths will raise exceptions if it is `None`).